qiskit_nature/second_q/algorithms/excited_states_solvers/qeom_vibrational_ops_builder.py

```diff
@@ -100,6 +101,13 @@ def _build_single_hopping_operator(
         label.append(f"-_{VibrationalOp.build_dual_index(num_modals, unocc)}")
 
     vibrational_op = VibrationalOp({" ".join(label): 1}, num_modals)
-    qubit_op: PauliSumOp = qubit_converter.convert_match(vibrational_op)
+
+    qubit_op: PauliSumOp
+    if isinstance(qubit_converter, QubitConverter):
+        qubit_op = qubit_converter.convert_match(vibrational_op)
+    elif issubclass(type(qubit_converter), QubitMapper):
+        qubit_op = qubit_converter.map_all(vibrational_op)
+    else:
+        raise QiskitNatureError("QubitConverter object is not valid")
```

Comment:
This should not be necessary given the type hint. As per my earlier comment a simple:
```python
if isinstance(..., QubitConverter):
    ...
else:
    ...
```
should suffice in all situations :+1:

Reply:
Also a file I forgot to update with the if/else syntax